sway-ir/src/parser.rs

```diff
@@ -59,16 +60,27 @@ mod ir_builder {
 
             rule contract() -> IrAstModule
                 = "contract" _ "{" _
-                  configs:init_config()* fn_decls:fn_decl()* "}" _
+                  configs:init_config()* _ global_vars:global_var()* _ fn_decls:fn_decl()* "}" _
                   metadata:metadata_decls() {
                     IrAstModule {
                         kind: crate::module::Kind::Contract,
                         configs,
+                        global_vars,
                         fn_decls,
                         metadata
                     }
                 }
 
+            rule global_var() -> IrAstGlobalVar
+                = m:("mut" _)? _ "global" _ name:path() _ ":" _ ty:ast_ty() _ "=" _ init:(op_const())? {
```

Comment:
For locals we have `local mut` and for globals now `mut global`. We should unify this in the same pattern. Personally, I am more for the `mut <local/global>` as done here, but more important is to have a consistent pattern.